scale/job/configuration/configurators.py

```diff
@@ -234,7 +234,7 @@ def __init__(self, workspaces):
                                  'SCALE_DB_PORT': db['PORT']}
         self._system_settings_hidden = {key: '*****' for key in self._system_settings.keys()}
 
-    def configure_scheduled_job(self, job_exe, job_type, interface):
+    def configure_scheduled_job(self, job_exe, job_type, interface, debug_level):
```

Comment:
I would rename `debug_level` to `logging_level`. Also be sure to add the new parameter to the docstring.